apps/next/src/pages/AccountManagement/AccountManagement.tsx

```diff
@@ -42,67 +38,50 @@ const dialogSlots: Pick<DialogProps, 'slots' | 'slotProps'> = {
   },
 };
 
-const dialogContentSx: SxProps = {
+export const dialogContentSx: SxProps = {
```

Comment:
If you need to use those things outside of `AccountManagement`, let's either move it out of this file, or just duplicate them (nothing wrong with that) -- especially since `dialogContentSx` references a container specific to account-management.

The way it is right now, you need to import `pages/AccountManagement/AccountManagement` from the lock screen, just for those props.